src/cmd/purge/purge.go

```diff
@@ -31,24 +27,6 @@ var purgeCmd = &cobra.Command{
 	RunE:  handleAllFolderPurge,
 }
 
-var mailCmd = &cobra.Command{
-	Use:   "mail",
-	Short: "Purges mail folders",
-	RunE:  handleMailFolderPurge,
-}
-
-var eventsCmd = &cobra.Command{
-	Use:   "events",
-	Short: "Purges calendar event folders",
-	RunE:  handleCalendarFolderPurge,
-}
-
-var contactsCmd = &cobra.Command{
-	Use:   "contacts",
-	Short: "Purges contacts folders",
-	RunE:  handleContactsFolderPurge,
-}
-
 var oneDriveCmd = &cobra.Command{
```

Comment:
@ashmrtn I left in your onedrive purge commands (though I'm not sure we're calling them atm) since we don't have pwsh coverage for those yet.  I'm hoping that if we investigate those controls we can get an actual hard delete, instead of needing to follow up in the UI manually.